### ISSUE TITLE
Interface does not fade away after mouse out in top/bottom of screen

### ISSUE THREAD

**Issue by tehsolace** [plexinc/plex-media-player#626]
While not in full-screen mode, play a video.  If you mouse-out of the top or bottom edges of the player's window, the interface will not fade away after few seconds (bug).  If you mouse-out of the left or right edges of the player's window, the interface will fade away (expected) as normal.

Tested in Windows 10 with current version of Plex Media Player 3.21.2.

**Comment by LongChair** [comment]
@tehsolace : could you confirm if this is still the case in current builds please ?

**Comment by DarkNemie** [comment]
As of Build (64-bit) 2.10.0.849-e02dbeca May 1, 2018 Running on windows 10 Pro x64 with Radeon 270x and Intel i7 - 4770k cpu The issue still exists. one thing to note is mousing out through the bottom UI to slowly or quickly seems to cause it more. Also this issue is worsened when PmP is used in a Multi monitor setup such as triple monitors with the PMP in the monitor that is above the other 2. Also this Issues exists when PMP is both in windowed mode and in full screen mode, I have not tested this behaviour in TV mode though.

**Comment by DarkNemie** [comment]
I have tested Build 12.10.0.854 that you posted on drop box. I can confirm in a Multi-Monitor setup where the previous versions failed to fade away, This one works great, the UI as well seems considerably more responsive as well. I have tested in windowed mode, fullscreen, windowed mode, Fullscreen mode. All are working as intended. The only thing I noticed is some times when moving the mouse into the plex monitor or window the UI will appear/disappear rapidly 3 or 4 times before staying, This does not affect anything as for usage just Thought it should be noted. Thank you. 

Also after a full 12 hours of usage I have not seen any other issues from the build.
